lib/observer/src/observer_html_lib.erl

```diff
@@ -147,7 +147,7 @@ stackdump_table(Tab,{Label0,Term0},Even, Colors) ->
                    %% Return address or catch tag. It is known to be a
                    %% flat list, shortish, possibly containing characters
                    %% greater than 255.
-                   href_proc_port(Term0)
+                   href_proc_port(lists:flatten(io_lib:format("~tp", [Term0])))
```

Comment:
Should it maybe be:
```
 stackdump_table(Tab,{Label0,Term0},Even, Colors) ->
     Label = io_lib:format("~ts",[Label0]),
     Term = case atom_to_list(Label0) of
                "y" ++ _ ->
                    %% Any term is possible, including huge ones.
                    all_or_expand(Tab,Term0);
                _ ->
                    %% Return address or catch tag. It is known to be a
                    %% flat list, shortish, possibly containing characters
                    %% greater than 255.
-                   href_proc_port(lists:flatten(io_lib:format("~tp", [Term0])))
+                   all_or_expand(Tab, Term0)
            end,
     tr(color(Even, Colors), [td("VALIGN=center",pre(Label)), td(pre(Term))]).
 ```
? (but then most of then change in 5fc8d39e2 would be undone, makes me hesitant)